COLLECTOR_VERSION

```diff
@@ -1 +1 @@
-3.7.3
+3.7.x-184-g9fa60cef05
```

Comment:
Should this be `slim`? Normal image take 2G whiel slim just 44M
https://quay.io/repository/rhacs-eng/collector?tab=tags

Reply:
The flavor of the image used is handled by stackrox, this is simply the version tag it uses as base and by default it should already be using the slim image.